### PR TITLE
EGN-570: Token security table view

### PR DESCRIPTION
**Pull request by 0xMasayoshi**
## PR-Codex overview
The focus of this PR is to enhance token security features and improve UI components across different packages.

### Detailed summary
- Added new token security checks and views
- Updated UI components in multiple packages
- Improved token import functionality in `TokenSelectorImportRow`

> The following files were skipped due to too many changes: `packages/wagmi/src/components/token-selector/TokenSelectorImportRow.tsx`, `packages/react-query/src/hooks/tokens/useTokenSecurity.ts`, `apps/evm/src/ui/swap/simple/simple-swap-token-not-found-dialog.tsx`, `apps/evm/src/ui/swap/cross-chain/cross-chain-swap-token-not-found-dialog.tsx`

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by chillichelli** [comment]
Awesome, mind showing a screenshot of what it looks like?

**Comment by 0xMasayoshi** [comment]
> Awesome, mind showing a screenshot of what it looks like?
image

image

**Comment by chillichelli** [comment]
Awesome looks great!

**Review by chillichelli — APPROVED** [review]

File: packages/react-query/src/hooks/tokens/useTokenSecurity.ts [hunk 1]
```diff
@@ -1,73 +1,197 @@
-import { useQuery } from '@tanstack/react-query'
+import { type UseQueryOptions, useQuery } from '@tanstack/react-query'
 import { ChainId } from 'sushi/chain'
 import { Token } from 'sushi/currency'
+import { z } from 'zod'
 
-const SUPPORTED_CHAINS: ChainId[] = [
+const SUPPORTED_CHAIN_IDS = [
```
Comment (LufyCZ) [inline packages/react-query/src/hooks/tokens/useTokenSecurity.ts:6]: This should probably be somewhere in the `sushi` pkg

Comment (0xMasayoshi) [reply packages/react-query/src/hooks/tokens/useTokenSecurity.ts]: moved to `sushi/config`

**Pull request merged**